sdk/src/wrapper.ts

```diff
@@ -0,0 +1,81 @@
+/**
```

Comment:
Can we use a better name for this file?

Reply:
Let's rename it to `store` and move client instantiation logic to `authenticate.tsx`. WDYT?

Reply:
Yes, let's do this.